### DISCUSSION TITLE
ConnectException's message is not transfered to crawlFailed

### DISCUSSION THREAD

**Discussion by jszczypk** [spatie/crawler#383] [Ideas]
Hi,

When ```ConnectException``` happens, in ```Spatie\Crawler\Handlers\CrawlRequestFailed``` it is converted to ```RequestException```. However exception's message is not transfered so in ```CrawlObserver::crawlFailed``` you do not have a reason why it really failed.
It happened to me when there was a timeout in Guzzle and message "cURL error 28: Operation timed out after 10001 milliseconds with 0 bytes received (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://(...)" was lost.

**Comment by freekmurze** [comment]
Feel free to submit a PR to improve the behaviour.
